### PR TITLE
Replace start command

### PR DESCRIPTION
**Pull request by TheOtterlord**
## Changes

- Replace `start` command from templates with `echo 'Warning: The `start` script is no longer included in Astro templates. Please use `dev` to start the development server.'`

## Testing

N/A

## Docs

https://github.com/withastro/docs/pull/9646

### CODE REVIEW

File: examples/basics/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "dev": "astro dev",
-    "start": "astro dev",
+    "start": "echo 'Warning: The `start` script is no longer included in Astro templates. Please use `dev` to start the development server.'",
```
Comment (matthewp) [inline examples/basics/package.json:8]: I'm in favor of just getting rid of these. Having warnings in *starter* templates feels a bit odd to me.

Comment (TheOtterlord) [reply examples/basics/package.json:8]: Sounds good. I stacked commits, so I can just revert the second one

**Comment by matthewp** [comment]
I think this should be against `next` and not `main`. Let's fix this in 5.0

**Comment by TheOtterlord** [comment]
Making a followup PR to prevent merging main into next

**Pull request closed without merging**